### ISSUE TITLE
loss very small， but F1 still low

### ISSUE THREAD

**Issue by brealisty** [jiangpinglei/Bert_ChinesewordSegment#7]
I fine-tune the model with my own data(about 500 items), beacuse data is small, so I don't part train data and dev data(train=dev) just for check my code right or not. The loss is very small(0.002-0.001), but the F1 score just 0.92.
